ecl/hql/hqlparse.cpp

```diff
@@ -1025,6 +1046,54 @@ void HqlLex::doLine(YYSTYPE & returnToken)
     }
 }
 
+
+bool HqlLex::readCheckNextToken(YYSTYPE & returnToken, int expected, unsigned errCode, const char * msg)
+{
+    if (yyLex(returnToken, false, 0) == expected)
+        return true;
+
+    reportError(returnToken, errCode, "%s", msg);
+    returnToken.release();
+    return false;
+}
+
+void HqlLex::doSlashSlashHash(YYSTYPE const & returnToken, const char * command)
+{
+    if (inmacro)
+    {
+        inmacro->doSlashSlashHash(returnToken, command);
+        return;
+    }
+
+    if (macroGathering)
+        return;
+
+    if (hasPrefix(command, "import", false))
+    {
+        const char * next = skipws(command + 6);
+        if (*next == '(')
+        {
+            next = skipws(next + 1);
+            const char * bra = strchr(next, ')');
+            if (bra)
+            {
+                StringAttr option(next, bra - next);
```

Comment:
Looks like you allow whitespace after the ( but not before the ), which seems a little inconsistent